.github/workflows/rgw-multisite-test/action.yml

```
@@ -86,8 +86,8 @@ runs:
         ns_name_secondary_object_zone='"rook-ceph-secondary/zone-b"' # double quotes intended
         tests/scripts/github-action-helper.sh restart_operator
         not_committed_msg="there are no changes to commit for RGW configuration period for CephObjectStore"
-        tests/scripts/github-action-helper.sh wait_for_operator_log_message "${not_committed_msg} ${ns_name_primary_object_zone}" 120
-        tests/scripts/github-action-helper.sh wait_for_operator_log_message "${not_committed_msg} ${ns_name_secondary_object_zone}" 120
+        tests/scripts/github-action-helper.sh wait_for_operator_log_message "${not_committed_msg} ${ns_name_primary_object_zone}" 180
```

What if we increase the timeout even more?
```suggestion
        tests/scripts/github-action-helper.sh wait_for_operator_log_message "${not_committed_msg} ${ns_name_primary_object_zone}" 240
```

---

it worked I set up to 60, I know is too much but works fine